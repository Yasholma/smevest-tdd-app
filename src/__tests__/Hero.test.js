import React from "react";
import { shallow } from "enzyme";
import Hero from "../components/hero/hero.component";

describe("<Hero /> Component", () => {
    const wrapper = shallow(<Hero />);

    test("should render hero component", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
