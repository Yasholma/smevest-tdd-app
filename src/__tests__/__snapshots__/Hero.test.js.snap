// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> Component should render hero component 1`] = `
<div
  className="Hero"
>
  <div
    className="GetStarted"
  >
    <h1>
      Welcome To Your Gateway Of Business Investment
    </h1>
    <Link
      className="Register"
      to="/register"
    >
      Get Started
    </Link>
  </div>
</div>
`;
